### PR TITLE
SAK-29478 Added sitestats visit integration

### PR DESCRIPTION
**Pull request by adrianfish**

### CODE REVIEW

File: roster2/src/java/org/sakaiproject/roster/tool/entityprovider/RosterSiteEntityProvider.java [hunk 1]
```diff
@@ -165,9 +167,32 @@ public Object getMembership(EntityReference reference, Map<String, Object> param
...
+            if (showVisits) {
+                sitePresenceTotals = sakaiProxy.getPresenceTotalsForSite(siteId);
+            }
+
+            boolean viewSiteVisits
+                = developerHelperService.isUserAllowedInEntityReference("/user/" + userId
+                                                    , RosterFunctions.ROSTER_FUNCTION_VIEWSITEVISITS
+                                                    , "/site/" + siteId);
+
+            System.out.println("viewSiteVisits: " + viewSiteVisits);
```
Comment (buckett) [inline roster2/src/java/org/sakaiproject/roster/tool/entityprovider/RosterSiteEntityProvider.java]: Looks like debugging information is still in here.

File: roster2/src/webapp/js/roster.js [hunk 2]
```diff
@@ -30,6 +30,15 @@
 	});
     
 	roster.i18n = $.i18n.map;
+
+    roster.i18n.months = roster.i18n.months.split(',');
+
+    /*
```
Comment (buckett) [inline roster2/src/webapp/js/roster.js]: Should this commented out coffee still be in there?

File: roster2/src/java/org/sakaiproject/roster/tool/entityprovider/RosterSiteEntityProvider.java (hunk 1 above)
Comment (adrianfish) [inline roster2/src/java/org/sakaiproject/roster/tool/entityprovider/RosterSiteEntityProvider.java]: You don't miss a trick you, do you? I'll take it out. Once, I left the word BALLS in a sysout and Steve S spotted it.

File: roster2/src/webapp/js/roster.js (hunk 2 above)
Comment (adrianfish) [inline roster2/src/webapp/js/roster.js]: It's not coffee, it's the loveliness of ECMA 5. No, it shouldn't be there though. I'll remove it forthwith :)

**Pull request merged**